### ISSUE TITLE
[FR] - Feature Request: Priority Sorting

### ISSUE THREAD

**Issue by ndharari** [Leonezz/obsidian-tasks-calendar-wrapper#56]
It would be good to add filtering by priority. If not, I cannot see what is of high priority and whats not on a given day.

**Comment by Leonezz** [comment]
agreed, actually a sorting combination system is under design, but it might take quite some time because I am busy with something else currently.

**Comment by ndharari** [comment]
Hi, thanks for your reply. One more thing, adding to the previous: group by heading!
This is something that the task plugin already allows, so maybe its not that difficult to implement.
